torchci/clickhouse_queries/weekly_force_merge_stats/query.sql

```diff
@@ -25,45 +25,60 @@ WITH issue_comments AS (
     FROM
         issue_comment FINAL
```

Comment:
Even though issue comments are technically a valid one to have FINAL run on, in practice they're edited so rarely that you can prob remove the FINAL and have only a negligible affect on the query output.

Might be worth it if there's a perf boost to be had (FINAL removal often gives a good one)

Reply:
yes, -98% memory by removing FINAL, and same output, thanks!